slices/sensible-utils.yaml

```diff
@@ -0,0 +1,27 @@
+package: sensible-utils
+
+essential:
+  - sensible-utils_copyright
+
+slices:
+  bins:
+    contents:
+      /usr/bin/select-editor:
+      /usr/bin/sensible-browser:
+      /usr/bin/sensible-editor:
+      /usr/bin/sensible-pager:
+      /usr/bin/sensible-terminal:
+      /usr/share/sensible-utils/bin/gettext:
```

Comment:
what about `/usr/share/sensible-utils/bin/realpath`? And, are these two really needed here or are they auxiliary binaries?

Reply:
Add the missing bin and moved them to their own slice